technology/overview.md

```diff
@@ -25,20 +25,26 @@ The following sections present four key technical layers that allow developers t
 
-As the head of a supported external chain updates, validators submit and confirm it within Pelagos’ DAG consensus process. Once a sufficient quorum is reached, this external block and all its data (state, events, balances, transactions) become available to all Pelagos Appchains as a native, trust-minimized data source.
+As the head of a supported external chain updates, validators submit and confirm it within Pelagos’ DAG consensus process. Once a sufficient quorum is reached, this external block and all its data (state, events, balances, transactions) become available to all Pelagos Appchains as a native, trust-minimized data source available for querying or data aggregation.
 
-The Pelagos consensus layer allows dApp builders to rely on the strong finality and fundamental security of the asset-managing blockchain that users' funds are native to, while enjoying the scalability and speed offered by the DAG consensus mechanisms. By limiting execution to consensus-approved events ensures that token issuance or state changes are valid, preserving atomicity and preventing double-spends or fraudulent minting.
+The Pelagos consensus layer enables dApp builders to leverage the strong finality and fundamental security assurances of the asset-managing blockchain that users' funds are native to, while enjoying the scalability and speed offered by the DAG consensus mechanisms. 
+
+By restricting execution to only those events that have been validated and agreed upon through the trust‑minimized consensus layer, Pelagos ensures that all token issuance and state changes are cryptographically verified and consensus‑approved. This strict validation prevents invalid or conflicting operations such as double‑spends or fraudulent minting. The consensus layer’s role in establishing a canonical, finalized order of events is essential for maintaining consistency and correctness. This finalized ordering can then be leveraged by higher‑level protocols to coordinate all‑or‑nothing execution across chains and state changes, enabling true atomic multichain transactions.
 
 > See more on the [validator role and responsibilities](./validating-appchain.md#validating-appchains-with-pelagos)
 
 ### Application logic layer
 
 The programmable application layer then leverages this high-availablity external blockchain data &mdash; supporting a full spectrum of business logic that can trigger automatic execution of predefined actions based on specified parameters.
 
-This ensures authentic, timely, and verifiable data delivery, powering rich business logic and seamless cross-chain composability. Furthermore, the application logic layer offers significant deployment flexibility.
+<!-- Ist't it a bit to broad? It doesn't give understanding what can be build and what kind of composability it gives.
```

Comment:
@JekaMas suggest that we expand of what can be built in the emergent section, use cases and link from there to those sections --> unless you have a high-level item you can suggest for here now?